### PR TITLE
ut: add timeout for race

### LINKED ISSUE #34892: Add timeout for race test

### PR DESCRIPTION
**Pull request by hawkingrei**
Signed-off-by: Weizhen Wang <wangweizhen@pingcap.com>

### What problem does this PR solve?

Issue Number: close #34892

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: tools/check/ut.go [hunk 1]
```diff
@@ -808,6 +808,8 @@ func (n *numa) testCommand(pkg string, fn string) *exec.Cmd {
 	if !race {
 		// Don't set timeout for race because it takes a longer when race is enabled.
```
Comment (tiancaiamao) [inline tools/check/ut.go:809]: Please also update comment

Comment (hawkingrei) [reply tools/check/ut.go]: Updated

**Review by wjhuang2016 — APPROVED** [review]

**Comment by hawkingrei** [comment]
@tiancaiamao PTAL

**Pull request merged**